scripts/presubmit.sh

```diff
@@ -112,9 +112,18 @@ main() {
   if [[ "${run_lint}" -eq 1 ]]; then
     check_cmd golangci-lint \
       'have you installed github.com/golangci/golangci-lint?' || exit 1
+    if ! type -p "prototool" > /dev/null; then
+      TMP="$(mktemp -d)"
+      trap 'rm -rf "${TMP}"' EXIT
+      cd "${TMP}"
+      GO111MODULE=on go get github.com/uber/prototool/cmd/prototool@v1.6
```

Comment:
This *might* be a better way of doing things, but it's inconsistent with all the existing tools and means that a tool installation failure is going to show up as part of the `script:` stage on Travis rather than the `install:` stage.  It also means that developers are going to end up with a tool in their `~/go/bin/` directory that doesn't have any source locally.

Reply:
I'll remove auto install to be consistent with the other tools

Reply:
still to do.